modular_citadel/code/modules/arousal/arousal.dm

```diff
@@ -21,16 +12,14 @@
 
 //Species vars
 /datum/species
-	var/arousal_gain_rate = AROUSAL_START_VALUE //Rate at which this species becomes aroused
-	var/arousal_lose_rate = AROUSAL_START_VALUE //Multiplier for how easily arousal can be relieved
 	var/list/cum_fluids = list("semen")
 	var/list/milk_fluids = list("milk")
 	var/list/femcum_fluids = list("femcum")
 
 //Mob procs
 /mob/living/carbon/human/proc/underwear_toggle()
 	set name = "Toggle undergarments"
-	set category = "IC"
+	set category = "Arousal"
```

Comment:
> More tabs.

I hate it just like the vore tab.

Reply:
moved to IC since there's only damn well 4 of 'em